packages/jsii-calc/package.json

```diff
@@ -36,11 +36,13 @@
   ],
   "dependencies": {
     "@scope/jsii-calc-base": "^0.8.0",
+    "@scope/jsii-calc-base-of-base": "^0.8.0",
```

Comment:
It suppresses a JSII warning because some of the types in there are re-exported... So it needs to be a peer, too... And peer deps should also be modeled in dependencies (or should they?)